CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [[v10.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v10.0.0...HEAD)] - 2020-xx-xx
 
-- Write your awesome change here (GH-xxxx)
+- **Breaking:** renamed eks security group id outputs and additional security group id inputs (by @slimm609)
```

Comment:
Please remove your change from the changelog.

Reply:
Why?   The module output is incorrect from what the terraform aws_eks_cluster is referencing and currently outputs.   Just 2 days ago, it was commented that it was a breaking change and to update it as so.